admin-dev/themes/new-theme/js/components/auto-complete-search.ts

```diff
@@ -29,26 +29,35 @@
  * functions which are, of course, overridable.
  */
 
+type DisplayFunction = (item: any) => string;
```

Comment:
```suggestion
type DisplayFunction = (item: any) => string;

```

Only about readability

Reply:
Done